README.md

```diff
@@ -1,5 +1,6 @@
 # AMP HTML ⚡ Validator
-Build status: [![build_status](https://travis-ci.com/ampproject/validator-java.svg?branch=master)](https://travis-ci.com/ampproject/validator-java)
+
+Build status: ![test](https://github.com/ampproject/validator-java/actions/workflows/test.yml/badge.svg?event=push)
```

Comment:
What would be the link to CI for diagnosing problems? We always logged into travis-ci.com/ampproject to investigate when we have a build failure.

Reply:
Surrounded the badge with a link to GitHub Actions